Swift/Tests/DocumentTest.swift

```diff
@@ -48,7 +48,7 @@ class DocumentTest: CBLTestCase {
         let array = MutableArrayObject()
         array.addValue("650-123-0001")
         array.addValue("650-123-0002")
-        doc.setValue(array, forKey: "array")
+        doc.setArray(array, forKey: "array")
```

Comment:
It's not consistent to use setArray here while the other places in this test use setValue.

Reply:
same here -  I thought interchanging them would prove they behave the same and as expected. Most of the tests have originally been made using setValue for ease of use/read.  

setValue is used in most of the other tests whenever we manipulate a document, so I can certainly change all these to use setArray.

Reply:
I realized this was a leftover... reverted